macros/models/_sector/tokens/transfers_enrich.sql

```diff
@@ -1,3 +1,9 @@
+{%- macro case_when_token_standard(native_column, erc20_column, else_column) -%}
```

Comment:
not sure if this is more readable or not but trying it

Reply:
I've recently learned we can also do
```
CASE token_standard
  WHEN 'native' THEN column1
  WHEN 'erc20' THEN column2
  ELSE column3
END
```
which is a bit more readable then all the inline comparisons